mods/cnc/rules/structures.yaml

```diff
@@ -6,15 +6,16 @@ FACT:
 	Tooltip:
 		Name: Construction Yard
 	Building:
-		Footprint: xxx xxx
-		Dimensions: 3,2
+		Footprint: xxx xxx ===
+		Dimensions: 3,3
+		LocalCenterOffset: 0,-420,0
```

Comment:
Actually you're right, these offsets don't take CameraPitch into consideration since they're not rotated like the TargetableOffsets are, so -512 is the correct value. Will fix it.